lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html

```diff
@@ -78,6 +78,14 @@
                 </adf-categories-management>
             </div>
         </ng-container>
+        <ng-container *ngFor="let customPanel of customPanels">
```

Comment:
I'm wondering and we can now make it shorter even more. This ng-container contains only single element (mat-expansion-panel) so probably ngFor can be set directly on mat-expansion-panel. If there is no customPanels (empty array) then just none mat-expansion-panel will be rendered. No need for ng-container when it conains only 1 element.

Reply:
Good point, changed